.github/workflows/ci.yml

```diff
@@ -9,8 +9,8 @@ name: CI
 
 on:
   push:
-    branches:
-      - master
+    branches-ignore:
+      - 'dependabot/**'
```

Comment:
Why this? My understanding is that it is useful to run checks against PRs opened by the dependabot.

Reply:
ok, my idea is to run build on all branch, it has ignored
when some body prepare PR do it on your fork in some bruch, so I want to run build in this case

I will remove this ignores - now it is not needed